File: Aula_23_03/Aluno.java
public class Aluno {
    private String nome;
    private String email;

    public Aluno(String nome) {
        this.nome = nome;
        this.email = gerarEmail(nome);
    }

    private String gerarEmail(String nome) {
        String[] dadosNome = nome.split(" ");
        return dadosNome[0].toLowerCase() + "." + dadosNome[dadosNome.length - 1].toLowerCase() + "@email.com";
    }

    public String getNome() { return nome; }
    public String getEmail() { return email; }
}

File: Aula_23_03/Aluno.py
class Aluno:
    def __init__(self, nome):
        self.nome = nome
        self.email = self.gerar_email(self.nome)
    
    def gerar_email(self, nome):
        dados_nome = nome.split()
        email = dados_nome[0].lower() + "." + dados_nome[-1].lower() + "@email.com"
        return email
    